scripts.js

```diff
@@ -225,6 +218,45 @@ window.adobeid = {
   locale: window.blog.language,
 };
 
+// Prep images for lazy loading and use adequate sizes
+let imgCount = 0;
+const observer = new MutationObserver(mutations => {
+  mutations.forEach(mutation => {
+    mutation.addedNodes.forEach(node => {
+      // only handle images with src=/hlx_*
+      if (node.tagName === 'IMG' && /\/hlx_/.test(node.src)) {
+        const img = node;
+        let width;
+        if (window.blog.pageType === window.blog.TYPE.TOPIC) {
+          // full width topic banner
+          width = window.innerWidth;
```

Comment:
We should probably lock topic banners at certain fixed sizes based on CSS breakpoints...

Reply:
i think anything that generates a broad number of image URLs should really be reconsidered, from a distributed caching standpoint i think this yields very undesirable results... i think we should probably lock in a few image sizes per image type and go with that.